app/models/alchemy/essence_headline.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Alchemy
+  class EssenceHeadline < BaseRecord
+    acts_as_essence(
+      ingredient_column: :body,
+      preview_text_method: :preview_text
```

Comment:
Since `acts_as_essence` is a mixin and you overwrote the method below, you can just remove this

```suggestion
```